### PR TITLE
add PrivacyInfo.xcprivacy to darwin frameworks.

### PR DESCRIPTION
**Pull request by cloudwebrtc**

### CODE REVIEW

File: sdk/objc/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyTrackingDomains</key>
+	<array/>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array/>
```
Comment (hiroshihorie) [inline sdk/objc/PrivacyInfo.xcprivacy]: It appears you haven't specified what type of APIs is used yet.

I think WebRTC uses [System boot time APIs](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278394)
```xml
	<key>NSPrivacyAccessedAPITypes</key>
	<array>
		<dict>
			<key>NSPrivacyAccessedAPIType</key>
			<string>NSPrivacyAccessedAPICategorySystemBootTime</string>
			<key>NSPrivacyAccessedAPITypeReasons</key>
			<array>
				<string>35F9.1</string>
				<string>8FFB.1</string>
			</array>
		</dict>
	</array>
```

Does it also use [Disk space APIs](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278394) ?

**Review by hiroshihorie — CHANGES_REQUESTED** [review]

File: sdk/objc/PrivacyInfo.xcprivacy (hunk 1 above)
Comment (hiroshihorie) [reply sdk/objc/PrivacyInfo.xcprivacy]: It appears to also be using stat() API, used to check file/folder existence:
Screenshot 2024-04-25 10 09 27

Discussion : https://forums.developer.apple.com/forums/thread/734750

So the final result can look like:

```xml
	<key>NSPrivacyAccessedAPITypes</key>
	<array>
		<dict>
			<key>NSPrivacyAccessedAPIType</key>
			<string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
			<key>NSPrivacyAccessedAPITypeReasons</key>
			<array>
				<string>C617.1</string>
			</array>
		</dict>
		<dict>
			<key>NSPrivacyAccessedAPIType</key>
			<string>NSPrivacyAccessedAPICategorySystemBootTime</string>
			<key>NSPrivacyAccessedAPITypeReasons</key>
			<array>
				<string>35F9.1</string>
				<string>8FFB.1</string>
			</array>
		</dict>
	</array>
```

Comment (hiroshihorie) [reply sdk/objc/PrivacyInfo.xcprivacy]: I'm not confident the NSPrivacyAccessedAPITypeReasons are correct, can you double check ? Thanks !

Comment (cloudwebrtc) [reply sdk/objc/PrivacyInfo.xcprivacy]: I compared here and the list you wrote looks right
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

**Review by davidliu — APPROVED** [review]

**Pull request merged**